--- a/init.d/auditd.conf
+++ b/init.d/auditd.conf
@@ -21,8 +21,8 @@
 action_mail_acct = root
 admin_space_left = 50
 admin_space_left_action = SUSPEND
-disk_full_action = SUSPEND
-disk_error_action = SUSPEND
+disk_full_action = SYSLOG
+disk_error_action = SYSLOG
 use_libwrap = yes
 ##tcp_listen_port = 60
 tcp_listen_queue = 5
--- a/audisp/plugins/syslog/syslog.conf
+++ b/audisp/plugins/syslog/syslog.conf
@@ -6,9 +6,9 @@
 # logged to. Valid options are LOG_LOCAL0 through 7, LOG_AUTH,
 # LOG_AUTHPRIV, LOG_DAEMON, LOG_SYSLOG, and LOG_USER.
 
-active = no
+active = yes
 direction = out
 path = /sbin/audisp-syslog
 type = always 
-args = LOG_INFO
+args = LOG_INFO LOG_LOCAL6
 format = string
